includes/abstracts/class-evf-form-fields.php

```diff
@@ -316,7 +316,7 @@ public function field_option( $option, $field, $args = array(), $echo = true ) {
 
 			// Field Meta. ---------------------------------------------------//
 			case 'meta':
-				$value   =  ! empty( $field['meta-key'] ) ? esc_attr( $field['meta-key'] ) : evf_get_meta_key_field_option( $field );
```

Comment:
This `evf_get_meta_key_field_option( $field );`  is used to get the meta key until form not being saved. Any reason for it's removal?